src/components/Dropdown/Dropdown.md

```diff
@@ -9,3 +9,8 @@ Allow user to validate a value by pressing `Enter`
 To hide the menu click on an item of the list or click outside to it
 The selected item has a specific design
 Items can be grouped with a title
+
+
+## Implementation
+
+To allow to use the dropdown with keyboard don't forget to add `e.stopPropagation();` on the onChange callback
```

Comment:
Shouldn't it be done directly in DropDown component ?

Reply:
Yes, it should be better, but I don't find the right place to add it without break everything

Reply:
I find a way 🤘